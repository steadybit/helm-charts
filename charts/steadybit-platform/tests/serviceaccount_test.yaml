templates:
  - serviceaccount.yaml
chart:
  appVersion: 0.0.0
  version: 0.0.0
tests:
  - it: should create service account using default values
    asserts:
      - matchSnapshot: {}
  - it: should create service account using custom name
    set:
      serviceAccount:
        name: custom-name
    asserts:
      - matchSnapshot: {}
  - it: should create service account using annotations
    set:
      serviceAccount:
        annotations:
          custome: value
    asserts:
      - matchSnapshot: {}
  - it: should not create service account
    set:
      serviceAccount:
        create: false
    asserts:
      - hasDocuments:
          count: 0