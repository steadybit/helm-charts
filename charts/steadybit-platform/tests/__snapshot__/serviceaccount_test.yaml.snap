should create service account using annotations:
  1: |
    apiVersion: v1
    kind: ServiceAccount
    metadata:
      annotations:
        custome: value
      labels:
        app.kubernetes.io/instance: RELEASE-NAME
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: steadybit-platform
        app.kubernetes.io/version: 0.0.0
        helm.sh/chart: steadybit-platform-0.0.0
      name: RELEASE-NAME-steadybit-platform
      namespace: NAMESPACE
should create service account using custom name:
  1: |
    apiVersion: v1
    kind: ServiceAccount
    metadata:
      annotations: {}
      labels:
        app.kubernetes.io/instance: RELEASE-NAME
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: steadybit-platform
        app.kubernetes.io/version: 0.0.0
        helm.sh/chart: steadybit-platform-0.0.0
      name: custom-name
      namespace: NAMESPACE
should create service account using default values:
  1: |
    apiVersion: v1
    kind: ServiceAccount
    metadata:
      annotations: {}
      labels:
        app.kubernetes.io/instance: RELEASE-NAME
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: steadybit-platform
        app.kubernetes.io/version: 0.0.0
        helm.sh/chart: steadybit-platform-0.0.0
      name: RELEASE-NAME-steadybit-platform
      namespace: NAMESPACE
